internal/venice-common/src/main/java/com/linkedin/venice/writer/VeniceWriter.java

```diff
@@ -319,6 +320,11 @@ public VeniceWriter(
                 + " is true and the topic is Version Topic");
       }
     }
+    if (maxRecordSizeBytes != -1 && maxSizeForUserPayloadPerMessageInBytes > maxRecordSizeBytes) {
+      throw new VeniceException(
+          MAX_SIZE_FOR_USER_PAYLOAD_PER_MESSAGE_IN_BYTES + '=' + maxSizeForUserPayloadPerMessageInBytes
+              + " cannot be set higher than " + MAX_RECORD_SIZE_BYTES + '=' + maxRecordSizeBytes);
```

Comment:
nit: What do you think of this kind of error message?
```
venice.writer.max.size.for.user.payload.per.message.in.bytes (<maxSizeForUserPayloadPerMessageInBytes>) cannot be set higher than venice.writer.max.record.size.bytes (<maxRecordSizeBytes>)
```

Reply:
Changed.

I'm used to the `=` for the tokenization / parsing of logs, but the `()` is totally fine too.